### PR TITLE
Port 9873 Add jenkins pipeline stages to jenkins integration

### PR DESCRIPTION
**Pull request by phalbert**
# Description

Updated docs for [adding the `stage` kind](https://github.com/port-labs/ocean/pull/960) in the Jenkins integration

## Updated docs pages

Please also include the path for the updated docs

- Jenkins (`docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md [hunk 1]
```diff
@@ -7,38 +7,45 @@ import TabItem from "@theme/TabItem"
...
-import PortApiRegionTip from "/docs/generalTemplates/_port_region_parameter_explanation_template.md"
-import OceanSaasInstallation from "/docs/build-your-software-catalog/sync-data-to-catalog/templates/_ocean_saas_installation.mdx"
+import AdvancedConfig from '../../../generalTemplates/\_ocean_advanced_configuration_note.md'
+import PortApiRegionTip from "/docs/generalTemplates/\_port_region_parameter_explanation_template.md"
+import OceanSaasInstallation from "/docs/build-your-software-catalog/sync-data-to-catalog/templates/\_ocean_saas_installation.mdx"
 
 # Jenkins
 
-Our Jenkins integration allows you to import `jobs`, `builds`, and `users` from your Jenkins environment into Port, according to your mapping and definitions.
+Our Jenkins integration allows you to import `jobs`, `builds`, `stages`, and `users` from your Jenkins environment into Port, according to your mapping and definitions.
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/jenkins/jenkins.md:16]:
```suggestion
Port's Jenkins integration allows you to import `jobs`, `builds`, `stages`, and `users` from your Jenkins environment into Port, according to your mapping and definitions.

```

**Review by PeyGis — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]
LGTM, has someone tested to see that the new snippets work as expected?

File: docs/build-your-software-catalog/sync-data-to-catalog/cicd/jenkins/jenkins.md [hunk 2]
```diff
@@ -774,6 +772,155 @@ resources:
...
+To prevent overwhelming your Ocean instance with potentially thousands of stages from Jenkins, the integration requires you to specify a specific job. This ensures that Ocean only retrieves stages related to that job, keeping things focused and efficient.
+
+**Important**: The integration will also fetch stages from all nested jobs within the specified job.
+:::
+
+```yaml showLineNumbers
+- kind: stage
+  selector:
+    query: 'true'
+    jobUrl: http://localhost:8080/job/limbopay/job/Limbo%20Core/job/main
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/cicd/jenkins/jenkins.md:857]: can we make it clear that the jobUrl is only an example so user's don't copy and paste this in their data sources mapping? And that they should replace it with their own jobUrl from one of their job

File: docs/build-your-software-catalog/sync-data-to-catalog/cicd/jenkins/jenkins.md [hunk 3]
```diff
@@ -774,6 +772,155 @@ resources:
...
+          stageUrl: env.OCEAN__INTEGRATION__CONFIG__JENKINS_HOST  + ._links.self.href
+        relations:
+          parentBuild: >-
+            ._links.self.href | sub("/execution/node/[0-9]+/wfapi/describe$";
+            "") | sub("^.*?/"; "") | gsub("%20"; "-") | gsub("%252F"; "-") |
+            gsub("/"; "-")
+- kind: stage
+  selector:
+    query: 'true'
+    jobUrl: http://localhost:8080/job/Phalbert/job/airframe-react
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/cicd/jenkins/jenkins.md:879]: same as above

File: docs/build-your-software-catalog/sync-data-to-catalog/cicd/jenkins/jenkins.md [hunk 4]
```diff
@@ -774,6 +772,155 @@ resources:
...
+          stageUrl: env.OCEAN__INTEGRATION__CONFIG__JENKINS_HOST  + ._links.self.href
+        relations:
+          parentBuild: >-
+            ._links.self.href | sub("/execution/node/[0-9]+/wfapi/describe$";
+            "") | sub("^.*?/"; "") | gsub("%20"; "-") | gsub("%252F"; "-") |
+            gsub("/"; "-")
+- kind: stage
+  selector:
+    query: 'true'
+    jobUrl: http://localhost:8080/job/Phalbert/job/autoshop_api
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/cicd/jenkins/jenkins.md:901]: same as above

**Review by PeyGis — CHANGES_REQUESTED** [review]
some comment

**Pull request closed without merging**